mass_editing/models/mass_editing_line.py

```diff
@@ -38,16 +38,8 @@ class MassEditingLine(models.Model):
         "'many2many_tags', 'selection'",
     )
 
-    @api.depends("field_id")
-    def _compute_widget_option(self):
-        # this function propose selection, depending on the field
-        for line in self.filtered("field_id"):
-            field = line.field_id
-            line.widget_option = False
-            if field.ttype == "many2one":
-                line.widget_option = "selection"
-            elif field.ttype == "many2many":
```

Comment:
The many2many_tags and the image one are useful, but not the selection one

Reply:
I think this is bad conceived, as this shouldn't be a computed field, but a stored one with "defaults". And please do it in a separate commit, as they are different changesets from the domain one.

Reply:
Try to define `default` in `widget_option` field but need to defined `field_id` previously, therefore it is now defined with an `@api.onchange`